app/lib/main/sharezone.dart

```diff
@@ -77,26 +78,28 @@ class _SharezoneState extends State<Sharezone> with WidgetsBindingObserver {
       textDirection: TextDirection.ltr,
       child: _ThemeSettingsProvider(
         blocDependencies: widget.blocDependencies,
-        child: Stack(
-          children: [
-            BlocProvider(
-              bloc: signUpBloc,
-              child: StreamBuilder<AuthUser>(
-                stream: listenToAuthStateChanged(),
-                builder: (context, snapshot) {
-                  if (snapshot.hasData) {
-                    widget.blocDependencies.authUser = snapshot.data;
-                    return SharezoneApp(widget.blocDependencies,
-                        Sharezone.analytics, widget.beitrittsversuche);
-                  }
-                  return AuthApp(
-                    blocDependencies: widget.blocDependencies,
-                    analytics: Sharezone.analytics,
-                  );
-                },
+        child: AlphaVersionBanner(
+          child: Stack(
```

Comment:
(Not really important, just general thinking...)

In my head from looking at the widget tree I'm thinking in the first second that this banner will be always shown.
I'm often times asking myself if it's nicer to hide the implementation like you've done here or explicitly write:  
```dart
        child: AlphaVersionBanner(
          isAlphaVersion: bool.fromEnvironment('ALPHA'),
          child: Stack(
```
On the other hand we can't do this for every widget anyways and in many cases it would probably convolute the code if the logic is more complex than this case.

Reply:
Changed it:
```dart
        child: AlphaVersionBanner(
          enabled: bool.fromEnvironment('ALPHA'),
          child: Stack(
```